test/functional/transactions/0.tx.js

```diff
@@ -0,0 +1,1001 @@
+
+			sendTransaction(transaction, function (err, res) {
+				node.expect(res).to.have.property('success').to.be.not.ok;
+				node.expect(res).to.have.property('message').to.equal('Invalid transaction amount');
+				badTransactions.push(transaction);
+				done();
+			});
+		});
+
+		it('when sender has NO funds should fail', function (done) {
+			var transaction = node.lisk.transaction.createTransaction('1L', 1, account.password);
+
+			sendTransaction(transaction, function (err, res) {
+				node.expect(res).to.have.property('success').to.be.not.ok;
+				node.expect(res).to.have.property('message').to.equal('Account does not have enough LSK: '+account.address+' balance: 0');
+				badTransactions.push(transaction);
+				done();
+			});
+		});
+
+		it('using entire balance should fail', function (done) {
+			var transaction = node.lisk.transaction.createTransaction(account.address, Math.floor(node.gAccount.balance), node.gAccount.password);
+
+			sendTransaction(transaction, function (err, res) {
+				node.expect(res).to.have.property('success').to.be.not.ok;
+				node.expect(res).to.have.property('message').to.match(/^Account does not have enough LSK: [0-9]+L balance: /);
+				badTransactions.push(transaction);
+				done();
+			});
+		});
+
+		it('sending funds from the genesis account should fail', function (done) {
+			var signedTransactionFromGenesis = {
+				type: 0,
+				amount: 1000,
+				senderPublicKey: 'c96dec3595ff6041c3bd28b76b8cf75dce8225173d1bd00241624ee89b50f2a8',
+				requesterPublicKey: null,
+				timestamp: 24259352,
+				asset: {},
+				recipientId: node.eAccount.address,
+				signature: 'f56a09b2f448f6371ffbe54fd9ac87b1be29fe29f27f001479e044a65e7e42fb1fa48dce6227282ad2a11145691421c4eea5d33ac7f83c6a42e1dcaa44572101',
+				id: '15307587316657110485',
+				fee: 10000000
+			};
+
+			sendTransaction(signedTransactionFromGenesis, function (err, res) {
+				node.expect(res).to.have.property('success').to.be.not.ok;
+				node.expect(res).to.have.property('message').equals('Invalid sender. Can not send from genesis account');
+				badTransactions.push(signedTransactionFromGenesis);
+				done();
+			});
+		});
+
+		it('when sender has funds should be OK', function (done) {
+			sendTransaction(goodTransaction, function (err, res) {
+				node.expect(res).to.have.property('success').to.be.ok;
+				node.expect(res).to.have.property('transactionId').to.equal(goodTransaction.id);
+				goodTransactions.push(goodTransaction);
+				done();
+			});
+		});
+
+		it('sending transaction with same ID twice should fail', function (done) {
+			sendTransaction(goodTransaction, function (err, res) {
+				node.expect(res).to.have.property('success').to.be.not.ok;
+				node.expect(res).to.have.property('message').to.equal('Transaction is already processed: '+goodTransaction.id);
+				done();
+			});
+		});
+	});
+
+	describe('confirmation', function () {
+
```

Comment:
Currently this is not the case in our eslint config, so this will change after #672.
@karmacoma Should we add a rule override to [eslint-config-lisk-base](https://github.com/LiskHQ/eslint-config-lisk-base)?